lib/jazzy/sourcekitten.rb

```diff
@@ -45,6 +45,10 @@ def self.make_doc_urls(docs, parents)
 
     # Run sourcekitten with given arguments and return STDOUT
     def self.run_sourcekitten(arguments)
+      unless Pathname('/Applications/Xcode.app').exist?
+        raise 'Xcode must be installed (or symlinked) at ' \
+        '/Applications/Xcode.app for jazzy to be able to work.'
+      end
       bin_path = Pathname(__FILE__).parent + 'sourcekitten/sourcekitten'
       command = "#{bin_path} #{(arguments).join(' ')}"
```

Comment:
BTW I'm totally new to the `jazzy` codebase, but is it normal that there is no `shellescape` there (instead of `join(' ')`)?